### PR TITLE
add: PR description template.

### PR DESCRIPTION
**Pull request by sayakpaul**
@LukeWood

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (LukeWood) [inline .github/PULL_REQUEST_TEMPLATE.md:24]: this is unfortunately not possible right now to check off before hand :(

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (LukeWood) [inline .github/PULL_REQUEST_TEMPLATE.md:54]: will this render in our new PRs or are comments excluded?

Comment (LukeWood) [reply .github/PULL_REQUEST_TEMPLATE.md:54]: ah, I assume it doesn't render.  Thats great, thanks!

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (sayakpaul) [reply .github/PULL_REQUEST_TEMPLATE.md:24]: Why?

**Review by LukeWood — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (sayakpaul) [reply .github/PULL_REQUEST_TEMPLATE.md:54]: This will render as comments.

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (LukeWood) [reply .github/PULL_REQUEST_TEMPLATE.md:24]: you can't update the keras.io docs from within a KerasCV PR.  I guess docstrings can be updated, but I still have to opt symbols into Keras-io

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (LukeWood) [reply .github/PULL_REQUEST_TEMPLATE.md:54]: gotcha, if so for now we can probably just remove lines 33 downward because we only have myself and Zhenyu really reviewing at the moment.  If we continue adding more reviewers this makes sense though, thanks!

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (sayakpaul) [reply .github/PULL_REQUEST_TEMPLATE.md:24]: What about the docstring that we include in the code?

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (sayakpaul) [reply .github/PULL_REQUEST_TEMPLATE.md:54]: I don't mind being tagged for reviewing model related PRs. But I understand if there are restrictions to that.

Comment (bhack) [reply .github/PULL_REQUEST_TEMPLATE.md:54]: IMHO It will scale better if we start to use the Github codeowners file:
https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (bhack) [inline .github/PULL_REQUEST_TEMPLATE.md:22]: I will try to underline that It is better or mandatory to open a ticket for a new feature and wait for a green light/approval before coding a PR.
In this way we will not waste contributors efforts if we don't want to allocate time to review or maintain that specific feature when the contributor will be MIA.

Comment (LukeWood) [reply .github/PULL_REQUEST_TEMPLATE.md:22]: It already is required in our community guidelines - but sometimes people miss it anyways.

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (LukeWood) [reply .github/PULL_REQUEST_TEMPLATE.md:54]: Let's  discuss in an Issue outside of this PR - this is a longer conversation and may end up blocking this PR for quite awhile.  better to track in its own issue

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (bhack) [reply .github/PULL_REQUEST_TEMPLATE.md:22]: Yes on that check box we could have a reminder

**Comment by sayakpaul** [comment]
@LukeWood pushed the changes.

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (bhack) [reply .github/PULL_REQUEST_TEMPLATE.md:36]: Probably when the project have merged a batch of their contributions at some point they could be ready to let scale the review system.

P.s. I've not started the thread just a contribution on a started one. Please open a new ticket or better a discussion if you are still interested.

**Comment by LukeWood** [comment]
Thanks @sayakpaul

**Pull request merged**
